src/css/components/Avatar.scss

```diff
@@ -0,0 +1,24 @@
+@import "../config/index.scss";
+
+// We're working with images, not margins so we scale this up
+$sizes: (
+  "Xs": $su-xsmall * 5,
+  "Sm": $su-small * 5,
+  "Md": $su-medium * 5,
+  "Lg": $su-large * 5,
+  "Xl": $su-xlarge * 5,
+  "Xxl": $su-xxlarge * 5
+);
+
+.Avatar {
+  @each $name, $value in $sizes {
+    &-#{$name} {
```

Comment:
According the [SUIT CSS naming conventions](https://github.com/suitcss/suit/blob/master/doc/naming-conventions.md) that TS-UI follows, a modifier class would use double dash with the modifier in camel case.  So I think we could just keep these lower case, so it would be `Avatar Avatar--xs`. 

Although maybe we could add the image `border-radius` on the `Avatar` component so that it is still a freestanding element with the base style applied. And the modifier could add the size. So like:

```
.Avatar {
  img {border-radius: 50%}
}

@each $name, $value in $sizes {
  .Avatar--#{$name} {
    width: $value;
    height: $value;
  }
}

Reply:
This comment is awesome! I learned so much from it!

1. I learned we use SUIT CSS naming conventions.
2. I learned that broder-radius can take a % value (I had no idea!)
3. I learned why we have the double dash in our class names sometimes!

Thanks @saralohr ! I've updated the code to better follow existing patterns!

Reply:
💯 😄